libs/gretty/src/main/groovy/org/akhikhl/gretty/ProjectUtils.groovy

```diff
@@ -10,6 +10,7 @@ package org.akhikhl.gretty
 import groovy.transform.CompileStatic
 import groovy.transform.TypeCheckingMode
 import org.apache.commons.io.FilenameUtils
+import org.apache.tools.ant.taskdefs.Java
```

Comment:
@yuezk would you mind removing the unused import you just introduced? Otherwise it looks good to go, @boris-petrov.

Reply:
Removed.